requestfinder.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018 SUSE LLC
```

Comment:
From what I understand the copyright and warranty at the top of every file is unnecessary (see "Berne Convention").

Reply:
nope, every single file should include copyright and license

Reply:
If one wants to be overly verbose perhaps, but it isn't.

Reply:
Even OBS doesn't. Can read about it at length, but at the most a single line reference to read `LICENSE` file. Every file is like putting the entire mailing address, name, and phone number embossed on every part of a consumer product. The convention is there and copyright granted automatically since 1800s. From my understanding the practice started as pandering to lawyers since common sense typically does not apply in that realm.